hknweb/events/models.py

```diff
@@ -1,66 +1,53 @@
     # event_type_id  = models.ForeignKey()
-    need_transportation = models.BooleanField(default=False)
-    view_permission_group_id = models.IntegerField(null=True)
-    rsvp_permission_group_id = models.IntegerField(null=True)
+    # need_transportation = models.BooleanField(default=False)
+    # view_permission_group_id = models.IntegerField(null=True)
+    # rsvp_permission_group_id = models.IntegerField(null=True)
     # view_permission_group_id = models.ForeignKey('people.Group', on_delete=models.SET_NULL, null=True)
     # rsvp_permission_group_id = models.ForeignKey('people.Group', on_delete=models.SET_NULL, null=True)
-    markdown    = models.BooleanField(default=False)
-    created_at  = models.DateTimeField(auto_now_add=True)
-    updated_at  = models.DateTimeField(auto_now=True)
+    # markdown    = models.BooleanField(default=False)
+    # created_at  = models.DateTimeField(auto_now_add=True)
+    # updated_at  = models.DateTimeField(auto_now=True)
 
     def __str__(self):
         return "Event(name={}, location={})".format(self.name, self.location)
 
 
 class Rsvp(models.Model):
-    NEED_RIDE = -1
-    HAVE_RIDE = 0
-    SM_SEDAN = 3
-    SEDAN = 4
-    MINIVAN = 6
-
-    TRANSPORT_ENUM = (
-        (NEED_RIDE, "I need a ride"),
-        (HAVE_RIDE, "Don't worry about me"),
-        (SM_SEDAN,  "I have a small sedan (4 seats)"),
-        (SEDAN,     "I have a sedan (5 seats)"),
-        (MINIVAN,   "I have a minivan (7 seats)"),
-    )
-
-    CONFIRMED = 't'
-    UNCONFIRMED = 'f'
-    REJECTED = 'r'
-
-    CONFIRMATION = (
-        (CONFIRMED,     "Confirmed"),
-        (UNCONFIRMED,   "Unconfirmed"),
-        (REJECTED,      "Rejected"),
-    )
-
-    id              = models.IntegerField(primary_key=True)
-    confirmed       = models.CharField(max_length=1,
-                                       choices=CONFIRMATION,
-                                       default=UNCONFIRMED)
-    confirm_comment = models.TextField()
-    person_id       = models.IntegerField(null=True)
-    # person_id       = models.ForeignKey('people.Person', on_delete=models.CASCADE)
-    event_id        = models.ForeignKey(Event, on_delete=models.CASCADE)
-    comment         = models.TextField()
-    transportation  = models.IntegerField(choices=TRANSPORT_ENUM,
-                                          default=HAVE_RIDE)
-    created_at      = models.DateTimeField(auto_now_add=True)
-    updated_at      = models.DateTimeField(auto_now=True)
+    # CONFIRMED = 't'
+    # UNCONFIRMED = 'f'
+    # REJECTED = 'r'
+
+    # CONFIRMATION = (
+    #     (CONFIRMED,     "Confirmed"),
+    #     (UNCONFIRMED,   "Unconfirmed"),
+    #     (REJECTED,      "Rejected"),
+    # )
+
+    # user  = models.ForeignKey(Profile, on_delete=models.CASCADE)
+    event = models.ForeignKey(Event, on_delete=models.CASCADE)
+    # id    = models.IntegerField(primary_key=True)
+    # confirmed       = models.CharField(max_length=1,
```

Comment:
Uh this needs a confirmed field? We want VP / officers to verify candidates showed up.
```py
confirmed = models.BooleanField(default=False)
```